realm/realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -65,39 +99,7 @@ public void generate() throws IOException, UnsupportedOperationException {
         writer.emitPackage(Constants.REALM_PACKAGE_NAME)
                 .emitEmptyLine();
 
-        ArrayList<String> imports = new ArrayList<String>();
-        imports.add("android.annotation.TargetApi");
-        imports.add("android.os.Build");
-        imports.add("android.util.JsonReader");
-        imports.add("android.util.JsonToken");
-        imports.add("io.realm.RealmObjectSchema");
-        imports.add("io.realm.RealmSchema");
-        imports.add("io.realm.exceptions.RealmMigrationNeededException");
-        imports.add("io.realm.internal.ColumnInfo");
-        imports.add("io.realm.internal.RealmObjectProxy");
-        imports.add("io.realm.internal.Row");
-        imports.add("io.realm.internal.Table");
-        imports.add("io.realm.internal.OsObject");
-        imports.add("io.realm.internal.SharedRealm");
-        if (!metadata.getBacklinkFields().isEmpty()) {
```

Comment:
I see.  So the generated code gets compiled and should not generate any lint warnings.

What would you think of a @SupressWarnings("all") at the head of each class?  If there *are* warnings, there is nothing that the user can do about them.  While I think that it would make a lot of sense for us to remove that @SupressWarniings in debug versions, so that *we* can make sure our code is good, perhaps it would be smartest to turn them off in release code.